app-backend/api/src/main/scala/annotation-project/AnnotationProjectPermissionRoutes.scala

```diff
@@ -141,4 +161,65 @@ trait AnnotationProjectPermissionRoutes
       }
     }
   }
+
+  def shareAnnotationProject(projectId: UUID): Route = authenticate { user =>
+    val shareCount =
+      AnnotationProjectDao
+        .getShareCount(projectId, user.id)
+        .transact(xa)
+        .unsafeToFuture
+    authorizeScopeLimit(
+      shareCount,
+      Domain.AnnotationProjects,
+      Action.Share,
+      user
+    ) {
+      entity(as[UserEmail]) { userByEmail =>
+        complete {
+          Auth0Service.getManagementBearerToken flatMap { managementToken =>
+            (for {
+              // Everything has to be Futures here because of methods in akka-http / Auth0Service
+              users <- UserDao
+                .findUsersByEmail(userByEmail.email)
+                .transact(xa)
+                .unsafeToFuture
+              // exist branch:
```

Comment:
This comment is confusing - the code below has both branches -- if a user exists and if one doesn't